docs/types/value-types.rst

```diff
@@ -66,16 +66,20 @@ Shifts
 ^^^^^^
 
 The result of a shift operation has the type of the left operand, truncating the result to match the type.
-Right operand must be unsigned type. Trying to shift by signed type will produce a compilation error.
+The right operand must be of unsigned type, trying to shift by an signed type will produce a compilation error.
 
-- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y``.
+- For positive and negative ``x`` values, ``x << y`` is equivalent to ``x * 2**y`` run in "unchecked" mode.
```

Comment:
Strictly speaking ``run in "unchecked" mode`` would need to be added to the ones below as well... ``x >> 300`` won't panic...

Reply:
maybe we should rephrase this altogether then and turn it into a mathematical expression instead of solidity code?